docs/scripts-creating.md

```diff
@@ -18,11 +18,91 @@ Scripts can be currently created from two sources:
 
 ### Create your first script from file (Postman Collection test)
 
-Let's assume you have exported Postman collection which want to test some internal service inside Kuberntertes cluster
-We've saved it into `~/Downloads/API-Health.postman_collection.json` 
+Create script json file:
```

Comment:
I don't think it's way to go with creating Postman collections - you're simply doing this in Postman not exporting script which you then cat to file? 

I would insert here screenshot from Postman rather than pasting script content